app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
Must be inherited from...?

Check 6 paragraph in "Programming Recommendations"
https://peps.python.org/pep-0008/#programming-recommendations